.github/workflows/build-wheel.yml

```diff
@@ -16,6 +16,9 @@ on:
       build_type:
         description: Value to set RUST_DAFT_PKG_BUILD_TYPE to (release, dev, nightly, etc.)
         type: string
+      use_new_name:
```

Comment:
alternately we can make a matrix of `package_name` that includes `[daft, getdaft, daft-lts]` rather that have two flags `[lts, use_new_name]`

Reply:
Yeah I was thinking about that too. I chose to add this new param because eventually we won't publish to getdaft, and at that point it would again make more sense to have only the lts parameter, so this is really just a temporary change